web/html/sectioning-elements/footer.md

```diff
@@ -19,7 +19,7 @@ aspects:
   - workout
 
 links:
-  - '[MDN: Footer](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer){website}'
```

Comment:
I think I like the resource description better how it was before. We don't need to say "docs" when we use the `documentation` tag.